pallets/file-system/src/utils.rs

```diff
@@ -2674,6 +2677,7 @@ where
                     &bucket_root,
                     &[(file_key, TrieRemoveMutation::default().into())],
                     &forest_proof,
+                    Some(bucket_id.encode()),
```

Comment:
we should have a common encode and decode function (decode function would also be exposed as a runtime api).